### DISCUSSION TITLE
Textual Inversion - from images (png/webp)

### DISCUSSION THREAD

**Discussion by creeduk** [Comfy-Org/ComfyUI#399] [Q&A]
Any way to use these or anyway to convert back to a pt file? I have a few including one of these early ones Victorian lace.

**Comment by WASasquatch** [comment]
Textual Inversions are "embeddings". Put them in your `/ComfyUI/models/embdeddings/` folder. 

`.bin` embdeddings from Diffusers can be renamed to `.pt` in WebUI without issues, I'm not sure about ComfyUI though. 

As far as training in ComfyUI, not yet, though would be cool to be able to train TIs and Hypernetworks, and LORAs.

**Reply by creeduk** [reply]
Weird I did put it there and it said not "warning, embedding:victorian-lace does not exist, ignoring" The pt files worked fine. I will double check and try again later today. T

**Reply by WASasquatch** [reply]
There is an issue with a comfy parsing I'd embeddings. I had to rename NT Negative to just "negativeembed" cause the `-` dashes in the name were messing it up I guess. @comfyanonymous should be aware of that.

**Reply by fractal-fumbler** [reply]
i think point of question is that to use embeddings that are in png formats
for example https://huggingface.co/Conflictx/Chempunk/blob/main/ChemPunk-96.png

as i understood atm ComfyUI didn't recognize such as embeddings

**Reply by WASasquatch** [reply]
> i think point of question is that to use embeddings that are in png formats for example https://huggingface.co/Conflictx/Chempunk/blob/main/ChemPunk-96.png
> 
> as i understood atm ComfyUI didn't recognize such as embeddings

Isn't that just PNGInfo prompt from WebUI? Not an embedding. That specifically is sample image from the training process, isn't it?

**Reply by fractal-fumbler** [reply]
no, it's not preview.
it's embedding. it can be used as embedding in a1111 webui.

**Reply by WASasquatch** [reply]
> no, it's not preview. it's embedding. it can be used as embedding in a1111 webui.

That's interesting, how? You just put it in embeddings folder?

**Reply by fractal-fumbler** [reply]
yes, same as pt or safetensor file
